frontend/src/helpers/helpers.ts

```diff
@@ -1,3 +1,18 @@
+import { dequal } from 'dequal';
```

Comment:
Considering we are only using it for JSON data, can we also use `dequal/lite`?

Reply:
BTW dequal seems like a rather nice package I did not know yet 👍

Reply:
Using `dequal/lite` now.